### PR TITLE
Solving the scroll issue in Shifting

### LINKED ISSUE #7487: Scroll issue in shifting in facilities.
**Issue**
In shifting in facilities, the scroll is common for all the fields like destination approved, transportation arranged, transferin progress , etc. This leads to an issue that if one field has more patients than others, the scroll for others goes below the max no if patients in that. If any one has 100 patients, others have 2 each, after viewing 100th patient of one, you need to scroll back up for the entire to view the ones for other fields. This is a UI issue and needs to be addressed.
@rithviknishad @nihal467 

issue

**Solution**
Separate scroll for each field like destination approved, transportation arranged, transfer-in progress , etc.

### PR DESCRIPTION
**Pull request by dwip708**
## Proposed Changes

- Fixes #7487
- Change 1
Added scroll for each div in shifting.
- Change 2
OverflowX made auto instead of none to allow navigating left and right for the different shifting options.

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Shifting/ShiftingBoard.tsx [hunk 1]
```diff
@@ -362,10 +362,10 @@ export default function ShiftingBoard({
     <div
       ref={drop}
       className={classNames(
-        "mr-2 h-full w-full flex-shrink-0 rounded-md bg-gray-200 pb-4 md:w-1/2 lg:w-1/3 xl:w-1/4",
+        "mr-2  w-full flex-shrink-0 rounded-md bg-gray-200 pb-4 md:w-1/2 lg:w-1/3 xl:w-1/4",
         isOver && "cursor-move"
       )}
-      style={{ minHeight: `${containerHeight + 100}px` }}
+      style={{ overflowY: "scroll" }}
```
Comment (rithviknishad) [inline src/Components/Shifting/ShiftingBoard.tsx:368]: use tailwindcss classes instead

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: src/Components/Shifting/ShiftingBoard.tsx (hunk 1 above)
Comment (dwip708) [reply src/Components/Shifting/ShiftingBoard.tsx:368]: > use tailwindcss classes instead

Okk I'll do that.

**Comment by nihal467** [comment]
@dwip708 what is the status on this PR

**Comment by dwip708** [comment]
> @dwip708 what is the status on this PR

As I said the main scroll issue is fixed but there is a pagination issue, and as a result only 14 records at max is shown under each card in shifting. I am working on that and will solve it by this weekend.

**Comment by nihal467** [comment]
@dwip708 closing this PR and unassigning you from this issue due to inactivity, feel free to reopen the same if you are still working on it

**Pull request closed without merging**